client/src/components/SignIn/SignIn.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import axios from "axios";
```

Comment:
1. Optional, move password recovery link right, and make it with opacity, and on hover opacity will be set to 1. See screenshot - https://pasteboard.co/Ifh03fT.png
2. Optional - make more space vertical between text and sign in. See screenshot - https://pasteboard.co/Ifh09hD.png
With these changes looks great!